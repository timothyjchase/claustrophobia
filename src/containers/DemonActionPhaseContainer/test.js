import React from 'react'
import { DemonActionPhaseContainer, mapStateToProps } from './index'

describe('<DemonActionPhaseContainer />', () => {
  describe('with basic props', () => {
    const props = {
      currentState: { scenarioKey: 'Test Scenario', demonsInPlay: 0 },
      completeDemonActionPhase: jest.fn(),
    }
    const component = shallow(<DemonActionPhaseContainer {...props} />)

    it('should match snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})

describe('mapStateToProps', () => {
  describe('with basic state', () => {
    const state = { current: {} }
    it('should return currentState', () => {
      expect(mapStateToProps(state)).toEqual({ currentState: {} })
    })
  })
})
