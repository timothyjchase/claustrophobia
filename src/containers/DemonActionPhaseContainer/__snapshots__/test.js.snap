// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DemonActionPhaseContainer /> with basic props should match snapshot 1`] = `
<DemonActionPhase
  completeDemonActionPhase={[MockFunction]}
  demonsInPlay={0}
  scenarioKey="Test Scenario"
  trogsSharpenedClaws={false}
  trogsSupernaturalSpeed={false}
/>
`;
